### ISSUE TITLE
How are the pre-trained weights trained?

### ISSUE THREAD

**Issue by cripsgreen** [HVision-NKU/Strip-R-CNN#20]
Hello, I would like to know which code was used to train the pre-trained weights and how the hyperparameters were set. Would you be able to share that with me?

**Comment by davidhuangal** [comment]
Could you please provide the hyper-parameters used to pre-train the StripNet backbones with `timm` ?

In the publication, 300 epochs for full training and 100 epochs for ablations are mentioned, but what about batch size, number of GPUs, learning rate, LR schedule, etc.? Or did you just use the defaults in `timm/train.py` ?
